common/v2/features/PurchaseMembership/components/ConfirmMembershipPurchaseMultiTx.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import * as R from 'ramda';
+
+import { ITxStatus, TxParcel } from 'v2/types';
+import { VerticalStepper } from 'v2/components';
+import { translateRaw } from 'v2/translations';
+import { IMembershipConfig } from '../config';
+import MembershipSelectedBanner from './MembershipSelectedBanner';
+
+import step1SVG from 'assets/images/icn-send.svg';
+import step2SVG from 'assets/images/icn-receive.svg';
+
+interface Props {
+  membershipSelected: IMembershipConfig;
+  currentTxIdx: number;
+  transactions: TxParcel[];
+  onComplete?(): void;
+}
+
+export default function ConfirmMembershipPurchase({
+  membershipSelected,
+  currentTxIdx,
+  transactions,
+  onComplete
+}: Props) {
+  const status = transactions.map(t => R.path(['status'], t));
+
+  const broadcasting = status.findIndex(s => s === ITxStatus.BROADCASTED);
```

Comment:
change this to be a boolean, so we don't end up having to interpret this being a number.

Reply:
It is supposed to be an index as it finds an index of a broadcasting tx, such that we make sure to show the step if the tx hasn't been mined yet.

Reply:
I more meant that since we're only using it to check if it `=== -1`, we could just swap this out for a boolean.